server/lib/src/migration_data/dl8/system_config.rs

```diff
@@ -0,0 +1,1073 @@
+use crate::constants::uuids::*;
+
+use crate::entry::{Entry, EntryInit, EntryInitNew, EntryNew};
+use crate::prelude::{Attribute, EntryClass};
+use crate::value::Value;
+
+// Default entries for system_config
+// This is separated because the password badlist section may become very long
+
+lazy_static! {
+    pub static ref E_SYSTEM_INFO_V1: EntryInitNew = entry_init!(
+        (Attribute::Class, EntryClass::Object.to_value()),
+        (Attribute::Class, EntryClass::SystemInfo.to_value()),
+        (Attribute::Class, EntryClass::System.to_value()),
+        (Attribute::Uuid, Value::Uuid(UUID_SYSTEM_INFO)),
+        (
+            Attribute::Description,
+            Value::new_utf8s("System (local) info and metadata object.")
+        ),
+        (Attribute::Version, Value::Uint32(20))
+    );
+    pub static ref E_DOMAIN_INFO_DL6: EntryInitNew = entry_init!(
+        (Attribute::Class, EntryClass::Object.to_value()),
+        (Attribute::Class, EntryClass::DomainInfo.to_value()),
+        (Attribute::Class, EntryClass::System.to_value()),
+        (Attribute::Class, EntryClass::KeyObject.to_value()),
+        (Attribute::Class, EntryClass::KeyObjectJwtEs256.to_value()),
+        (Attribute::Class, EntryClass::KeyObjectJweA128GCM.to_value()),
+        (Attribute::Name, Value::new_iname("domain_local")),
+        (Attribute::Uuid, Value::Uuid(UUID_DOMAIN_INFO)),
+        (
+            Attribute::Description,
+            Value::new_utf8s("This local domain's info and metadata object.")
+        )
+    );
+    pub static ref E_SYSTEM_CONFIG_V1: EntryInitNew = entry_init!(
+        (Attribute::Class, EntryClass::Object.to_value()),
+        (Attribute::Class, EntryClass::SystemConfig.to_value()),
+        (Attribute::Class, EntryClass::System.to_value()),
+        (Attribute::Uuid, Value::Uuid(UUID_SYSTEM_CONFIG)),
+        (
+            Attribute::Description,
+            Value::new_utf8s("System (replicated) configuration options.")
+        ),
+        (
+            Attribute::BadlistPassword,
+            Value::new_iutf8("bad@no3IBTyqHu$list")
+        ),
+        (
+            Attribute::BadlistPassword,
+            Value::new_iutf8(
+                "demo_badlist_shohfie3aeci2oobur0aru9uushah6EiPi2woh4hohngoighaiRuepieN3ongoo1"
+            )
+        ),
+        (
+            Attribute::BadlistPassword,
+            Value::new_iutf8("100preteamare")
+        ),
+        (Attribute::BadlistPassword, Value::new_iutf8("14defebrero")),
+        (Attribute::BadlistPassword, Value::new_iutf8("1life1love")),
+        (Attribute::BadlistPassword, Value::new_iutf8("1life2live")),
+        (Attribute::BadlistPassword, Value::new_iutf8("1love1life")),
+        (Attribute::BadlistPassword, Value::new_iutf8("1love4life")),
+        (Attribute::BadlistPassword, Value::new_iutf8("212224236248")),
+        (Attribute::BadlistPassword, Value::new_iutf8("2813308004")),
```

Comment:
Can we codegen these from a common file? during build read all lines from the file and turn each into 1 attribute?

Reply:
Probably but that should be it's own PR I think. That's it's own block of work.

Reply:
Yeah easy. Not wanting it in this PR. Just want it done eventually